src/Features/CSharp/Portable/CSharpFeaturesResources.resx

```diff
@@ -378,6 +378,30 @@
     <value>into clause</value>
     <comment>{Locked="into"} "into" is a C# keyword and should not be localized.</comment>
   </data>
+  <data name="is_pattern" xml:space="preserve">
+    <value>is pattern</value>
+    <comment>{Locked="is"} "is" is a C# keyword and should not be localized.</comment>
+  </data>
+  <data name="deconstruction" xml:space="preserve">
+    <value>deconstruction</value>
+  </data>
+  <data name="tuple" xml:space="preserve">
+    <value>tuple</value>
+  </data>
+  <data name="local_function" xml:space="preserve">
+    <value>local function</value>
+  </data>
+  <data name="out_var" xml:space="preserve">
+    <value>out variable</value>
+    <comment>{Locked="out"} "out" is a C# keyword and should not be localized.</comment>
+  </data>
+  <data name="ref_local_or_expression" xml:space="preserve">
+    <value>ref local or expression</value>
+    <comment>{Locked="ref"} "ref" is a C# keyword and should not be localized.</comment>
+  </data>
+  <data name="v7_switch" xml:space="preserve">
+    <value>C# 7 enhanced switch statement</value>
```

Comment:
I don't think it will need to be updated. C# 7 introduces an upgraded switch statement, which can have the same syntax but accept more types in the `switch(...)`.

@gafter Is there a more descriptive name I could use to describe the v7 switch, by any chance?